src/components/virtualRepeat/virtualRepeater.js

```diff
@@ -22,7 +22,9 @@ angular.module('material.components.virtualRepeat', [
  *
  * @usage
  * <hljs lang="html">
- * <md-virtual-repeat-container></md-virtual-repeat-container>
+ * <md-virtual-repeat-container>
+ *   <div md-virtual-repeat="i in items" md-item-size="10">Hello {{i}}!</div>
+ * </md-virtual-repeat-container>
```

Comment:
@ThomasBurleson, I think `mdItemSize` is the size (width/height) of each item (in pixels), not the number of visible items.

Regarding the 2 examples, I don't find it super-useful either (I just copied it from the `virtualRepeat` which has the same 2 examples. (TBH, I don't think horizontal scrolling of many elements is all that common, so leaving `mdOrientHorizontal` out of the example wouldn't bother me much :)).